### PR TITLE
Adjust Underlying Science doc23 section

### PR DESCRIPTION
**Pull request by CatalinaYepes**

### CODE REVIEW

File: doc23/underlying-science/overview.rst [hunk 1]
```diff
@@ -1,62 +1,12 @@
...
-following link:
-`https://github.com/gem/oq-engine. <https://github.com/gem/oq-engine>`__
-
-.. figure:: _images/schematic_oq_suite.jpeg
-
-   *A schematic describing the OpenQuake suite.*
 
 Structure of the OpenQuake engine
-*********************************
+--------------------------------
```
Comment (nastasi-oq) [inline doc23/underlying-science/overview.rst:9]:
```suggestion
---------------------------------
```

File: doc23/underlying-science/overview.rst [hunk 2]
```diff
@@ -1,62 +1,12 @@
-Introduction
-============
+Overview of the OpenQuake engine
+=================================
```
Comment (nastasi-oq) [inline doc23/underlying-science/overview.rst:2]:
```suggestion
================================
```

**Review by nastasi-oq — COMMENTED** [review]
Just a couple of misaligned underlines, apart from them LGTM

**Review by raoanirudh — COMMENTED** [review]
LGTM

**Pull request merged**